features/step_definitions/show_dishes_steps.rb

```diff
@@ -0,0 +1,21 @@
+Given("the following dishes exists") do |table|
+  table.hashes.each do |hash|
+    Dish.create!(hash)
+  end
+end
+
+Given("I visit the dish page") do
+  visit '/restaurants/show'
+end
+
+Then("I should see the text {string}") do |text|
+  expect(page).to have_content text
+end
+
+Then("I should see the picture {string}") do |pic_url|
+  expect(page).to have_xpath("//img[@src='#{pic_url}']")
```

Comment:
Are you sure there is no matcher for this in Capybara? Xpath is okay but not very readable.

Reply:
Not sure at all.